contracts/cctp/SynapseCCTP.sol

```diff
@@ -133,7 +143,7 @@ contract SynapseCCTP is SynapseCCTPEvents, ISynapseCCTP {
         address token = _getLocalMintedToken(originDomain, originBurnToken);
         uint256 fee;
         // Apply the bridging fee. This will revert if amount <= fee.
-        (amount, fee) = _applyFee(token, amount);
+        (amount, fee) = _applyRelayerFee(token, amount, requestVersion == RequestLib.REQUEST_SWAP);
```

Comment:
Should the relayers be whitelisted/authenticated in any way?

Reply:
Good question! The contracts are supposed to allow any relayers to complete the tx (as proving invalid data simply leads to tx being reverted). However, I understand that having the ability to toggle such things wound not hurt. And this could be extended to contracts being paused as well.

Therefore, I feel that having these toggles might be helpful:
- Sending tokens: paused/unpaused
- Receiving tokens: paused/whitelisted-relayer-only/any-releyers-allowed

@aureliusbtc would appreciate your opinion here as well.

Reply:
To be resolved in the upcoming PR